spec/controllers/graphql/queries/freaks_spec.rb

```diff
@@ -29,6 +27,17 @@ module Graphql
 
       it { is_expected.to match_response_for(query: :freaks, sample: :freaks_pagination) }
     end
+
+    context 'when advanced freak was created' do
+      it 'returns a Advanced freak' do
```

Comment:
Nothing happens within the context. You could take the test out and it would work exactly the same.

The setup within contexts is supposed to be different, unless you want to describe conceptually a context for ease of readability.